libraries/Wire/doc/i2c_debugging.md

```diff
@@ -0,0 +1,267 @@
+# Debugging I2C
+
+With the release of Arduino-ESP32 V1.0.0 the I2C subsystem contains code to exhaustively report communication errors.  
```

Comment:
It is valid for all version since V1.0.0rc3 .

Reply:
the way I understand it is that in v 1.0.0 this is fixed, not that it's fixed starting from rc3 or whatever later on. Maybe change to 1.0.1 to avoid confusion?